exercises/twelve-days/canonical-data.json

```diff
@@ -0,0 +1,96 @@
+{
+  "exercise": "twelve-days",
+  "version": "1.0.0",
+  "cases": [
+    {
+      "description": "returns verse number 1",
```

Comment:
I'd like to see more descriptive descriptions. This does not tell me anything that cannot be automatically generated from the input.

How about:
"The first day's gift is a partridge in a pair tree"
"The second day's gift is two turtle doves"
etc.